packages/babel-helpers/src/helpers/wrapRegExp.js

```diff
@@ -31,7 +31,8 @@ export default function _wrapRegExp() {
         this,
         str,
         substitution.replace(/\$<([^>]+)>/g, function (_, name) {
-          return "$" + groups[name];
+          var group = groups[name];
+          return "$" + (typeof group === "number" ? group : group.join("$"));
```

Comment:
```suggestion
          return "$" + (Array.isArray(group) ? group.join("$") : group);
```
Using `Array.isArray` saves us 3 bytes.